webapp/src/shared/context/GamePairingContext.tsx

```diff
@@ -115,44 +128,62 @@ export const GamePairingProvider = ({ children }: { children: ReactNode }) => {
               },
               {
                 onClick: () => {
-                  socket.emit('gameChallengeResponse', {
-                    gameRoomId,
-                    status: GameChallengeStatus.CHALLENGE_DECLINED,
-                  } as GameChallengeResponseDto);
+                  socket.emit(
+                    gameQueueClientToServerWsEvents.gameChallengeResponse,
+                    {
+                      gameRoomId,
+                      status: GameChallengeStatus.CHALLENGE_DECLINED,
+                    } as GameChallengeResponseDto,
+                  );
                   setGameCtx({
                     ...gameCtx,
                     isWaitingToPlay: false,
                   });
+                  // noop navigation
```

Comment:
leftover?

Reply:
Not really. I'm explicitly stating in a comment something that may not be obvious to the programmer that casually bumps into this, and that may believe it's missing.

I may rather expand it with a longer comment explaining this. I'll do it.

Reply:
Appreciate it. What is the meaning of `noop navigation`?

Reply:
> In computer science, a NOP, no-op, or NOOP is a machine language instruction and its assembly language mnemonic, programming language statement, or computer protocol command that does nothing.

Which means that this sequence of code is not triggering any navigation, but contrary to what could be intuitively assumed (at least by my intuition) this is the expected and intended behaviour.